src/main/java/me/jellysquid/mods/sodium/mixin/features/render_layer/MixinRenderLayers.java

```diff
@@ -29,4 +36,11 @@
         BLOCKS = new Reference2ReferenceOpenHashMap<>(BLOCKS);
         FLUIDS = new Reference2ReferenceOpenHashMap<>(FLUIDS);
     }
+    @Inject(method = "getBlockLayer(Lnet/minecraft/block/BlockState;)Lnet/minecraft/client/render/RenderLayer;", at = @At(value = "RETURN"), cancellable = true)
```

Comment:
Can we move this injection into `features.render_layer.leaves` package as well? I don't want to mix bug fixes with the existing optimizations here.